integration-tests/smoke/ccip/ccip_gas_price_updates_test.go

```diff
@@ -13,17 +12,14 @@ import (
 	"github.com/smartcontractkit/chainlink-common/pkg/config"
 	"github.com/smartcontractkit/chainlink-common/pkg/utils/tests"
 
-	"github.com/smartcontractkit/chainlink/deployment"
 	"github.com/smartcontractkit/chainlink/deployment/ccip/changeset"
 	"github.com/smartcontractkit/chainlink/deployment/ccip/changeset/testhelpers"
 	"github.com/smartcontractkit/chainlink/deployment/ccip/changeset/v1_6"
 	testsetups "github.com/smartcontractkit/chainlink/integration-tests/testsetups/ccip"
-	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/v1_6_0/fee_quoter"
 )
 
-// Test_CCIPGasPriceUpdates tests that chain fee price updates are propagated correctly when
-// price reaches some deviation threshold or when the price has expired.
-func Test_CCIPGasPriceUpdates(t *testing.T) {
+// Test_CCIPGasPriceUpdates tests that chain fee price updates are propagated correctly when expiry time is reached.
+func Test_CCIPGasPriceUpdatesWriteFrequency(t *testing.T) {
```

Comment:
I think it breaks the CI configuration because the test names are different now.


```yaml
  - id: smoke/ccip/ccip_gas_price_updates_test.go:*
    path: integration-tests/smoke/ccip/ccip_gas_price_updates_test.go
    test_env_type: docker
    runs_on: ubuntu-latest
    triggers:
      - PR E2E Core Tests
      - Nightly E2E Tests
    test_cmd: cd integration-tests && go test github.com/smartcontractkit/chainlink/integration-tests/smoke/ccip -v -run ^Test_CCIPGasPriceUpdates$ -timeout 18m -count=1 -test.parallel=1 -json
    pyroscope_env: ci-smoke-ccipv1_6-evm-simulated
    test_env_vars:
      E2E_TEST_SELECTED_NETWORK: SIMULATED_1,SIMULATED_2
      E2E_JD_VERSION: 0.9.0
      CCIP_V16_TEST_ENV: docker
```

Reply:
this part enforces the end of the string after `Updates`
```-v -run ^Test_CCIPGasPriceUpdates$```